code-coverage.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
A comment describing what -e does like you did on line 28 would help, first thing i did when looking at this file was lookup what `set -e` did